packages/control-property-editor/src/components/ThemeSelectorCallout.scss

```diff
@@ -46,6 +46,16 @@
     background-color: var(--vscode-progressBar-background);
 }
 
+.theme-child:hover {
+    background-color: var(--vscode-toolbar-hoverBackground, var(--vscode-menubar-selectionBackground));
+    outline: 1px dashed var(--vscode-contrastActiveBorder);
+}
+
+.selected:hover {
+    border-radius: 1px;
+    background-color: var(--vscode-progressBar-background);
```

Comment:
This is duplicating styles from line 44-47. Is this to avoid styles being overwritten by selector `.theme-child:hover` styles? If that is the case then we can use [not pseudo class](https://developer.mozilla.org/en-US/docs/Web/CSS/:not), to avoid altering `selected` styles even when hover is active.

Reply:
Yes, I need to avoid overwriting `.theme-child:hover` to prevent the selected theme from having a different background color when hovering. If I didn't, the selected theme would show the background color meant for hovering, rather than the selected background color. I will try to do it with using not pseudo class. Thanks!